gitlab-pipeline/stage/init-mender-monitor-package.yml

```diff
@@ -0,0 +1,27 @@
+init:mender-monitor-package:
+  stage: init
+  image: alpine:3.12
+  before_script:
+    - apk add --no-cache git openssh
+    # Prepare SSH keys
+    - eval $(ssh-agent -s)
+    - echo "$SSH_PRIVATE_KEY" | tr -d '\r' | ssh-add - > /dev/null
+    - mkdir -p ~/.ssh
+    - chmod 700 ~/.ssh
+    - ssh-keyscan github.com >> ~/.ssh/known_hosts
+    # Clone monitor-client
+    - git clone git@github.com:mendersoftware/monitor-client monitor-client
+    - cd monitor-client
+    - ( git fetch -u -f origin $MONITOR_CLIENT_REV:pr &&
+          git checkout pr ||
+          git checkout -f -b pr $MONITOR_CLIENT_REV
+      ) || return 1
+  script:
+    - apk add --no-cache make git
```

Comment:
Question:
Why add `git` twice?
And why not do all these installs in the `before_script`?

Reply:
Because `before_script` is overridden in [`monitor-client` pipeline](https://github.com/mendersoftware/monitor-client/pull/31/commits/4133424e3de2cdde26238c05fb8d93a99fcba640#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R26).

Reply:
That explains it :)

Reply:
I know is not ideal. Thanks for paying attention ;)